scalableminds/webknossos
# Refactor utils

This PR combines the template_helpers and format_utils classes as they are basically the same. It also refactors the resulting format_utils class and the utils class to use exported functions instead of some class or object that contains those functions. I've removed unused methods while I was at it.

### URL of deployed dev instance (used for testing):
- https://refactorutils.webknossos.xyz

### Steps to test:
- Do a basic tracing view test. This in combination with a successful CI run should be enough.

### Issues:
- fixes #3004

------
